bcda/client/bluebutton.go

```diff
@@ -29,9 +29,9 @@ var logger *logrus.Logger
 const blueButtonBasePath = "/v1/fhir"
 
 type APIClient interface {
-	GetExplanationOfBenefitData(patientID, jobID string) (string, error)
-	GetPatientData(patientID, jobID string) (string, error)
-	GetCoverageData(beneficiaryID, jobID string) (string, error)
+	GetExplanationOfBenefitData(patientID, jobID string, cmsID string) (string, error)
```

Comment:
```suggestion
	GetExplanationOfBenefitData(patientID, jobID, cmsID string) (string, error)
```
Extra `string` in there.

What do you think about renaming `cmsID` to `acoCMSID` (or `cmsACOID`) everywhere? I can imagine someone wondering if it means an ACO ID or perhaps some beneficiary ID.

Reply:
couldn't agree more! I think we can safely rename the cmsID --> acoCMSID but I believe the header key has to stay the same (BCDA-CMSID). This is what BB is expecting from us

Reply:
@embh actually after looking at that name change, we use cmsID everywhere, I found 12 different files. I think if we do it, then we should do it do it. What do you think about making that a separate ticket?

Reply:
I see some uses of `cmsID`, but they look to be on the ACO model or functions that are related to ACOs (e.g., `GetACOByCMSID()`. I don't think those need to change.

Reply:
oh okay, so just the new parameter additions??

Reply:
Yes, sorry about that! My original comment was confusing and should have said "everywhere *in this PR*." :)

Reply:
I actually think this is fine for now - 2 reasons mostly.
1) Based on your earlier comment: "This has me wondering yet again if we should keep our internal ACO UUIDs now that we have the CMS IDs." I think this is valid and the cmsID can just be our aco id or we can decide the name at a later date. I think to do the name change now from  cmsID --> acoCMSID/cmsACOID doesn't really make a lot of sense if this is something we're really considering.
2) Also the name change really needs to be done in all locations of our code not just this pr, lets be consistent and lock these names down, not just this one but also the beneficiary id vs. patient id thing. We should have a ticket to update all of this.

Reply:
@tbellj Closing the loop here.  Can you make a ticket to address the out-of-scope concerns?